specs/Environment/AirQualityObserved/doc/spec.md

```diff
@@ -139,7 +139,8 @@ measurand, ex. CO. The structure of such an attribute will be as follows:
         -   Type: [Text](https://schema.org/Text)
         -   Mandatory
     -   `description` : short description of the measurand
-        -   Type: [Text](https://schema.org/Text)
```

Comment:
please double check if this description applies here

Reply:
The description of the measurand is still a description and the definition of `schema.org/description` still applies:

> A description of the item.

Yes this is metadata (or *Property-of-a-Property* in NGSI-LD terms), but the *Property-of-a-Property* still has the name `description` which would expand to `https://uri.etsi.org/ngsi-ld/description` under NGSI-LD so I think it is still valid.

Reply:
you are totally right!